### ISSUE TITLE
Feedback on tool cards accessability

### ISSUE THREAD

**Issue by camfindlay** [opendataboardgame/game#35]
Something interesting I've just come across while trying to play... I printed some black & white paper copies to try it out. It's difficult to determine on each tool card what impact score is available when I complete a tool. I could imaging this might be the experience for people with certain colourblindness or visual impairment.

I'd suggest either:
- Only including the impacts relevant on the tool card (rather than gray irrelevant ones) or
- Increase the contrast (make the gray very very faint and the colour very bold) and/or
- Perhaps a simple line under the impact items that apply for the tool

Example:

![img_20160521_075046706](https://cloud.githubusercontent.com/assets/367847/15440354/25f9f5ec-1f29-11e6-9652-96b5203150c5.jpg)

**Comment by Stephen-Gates** [comment]
Agreed. Also confusing on colour printed cards. Suggest the symbol is removed from the card.

**Comment by ellenbroad** [comment]
@Stephen-Gates do you mean even on the colour printed cards it's hard to see which is black and which is shaded grey?

**Comment by Stephen-Gates** [comment]
@ellenbroad Yes. Using the colour cards we did stop and have to compare cards to clarify that. 

It may just be one of those "first time played" issues". 

![img_6903 copy](https://cloud.githubusercontent.com/assets/9379524/17678563/fc82fd76-637a-11e6-81e4-138a6591ac48.jpg)
